### PR TITLE
fix(TDI-49411): new tck jdbc for studio and cloud

### PR DESCRIPTION
**Pull request by wwang-talend**

### CODE REVIEW

**Comment by wwang-talend** [comment]
no preparestatement usage for ddl

File: newjdbc/jdbc.ipr [hunk 1]
```diff
@@ -0,0 +1,1300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (jzhao-talend) [inline newjdbc/jdbc.ipr:2]: don't forget remove .ipr & .iws files

**Review by jzhao-talend — APPROVED** [review]

**Comment by wwang-talend** [comment]
have checked all sonar issues, now stauts is ok. And integration tests not detected for coverage, will check later.

**Comment by wwang-talend** [comment]
https://jenkins-connectors.datapwn.com/job/connectors%20se/job/build/job/PR-1129/build?delay=0sec

**Pull request merged**